src/dtflw/input_table.py

```diff
@@ -21,10 +22,27 @@ def abs_file_path(self) -> str:
         Returns absolute file path of the table, or None if no suitable table was found.
         """
         return self.__abs_file_path
+    
+    @property
+    def is_optional(self) -> bool:
+        """
+        Returns the optionality indicator of the table.
+        """
+        return self.__is_optional
 
     def needs_eval(self) -> bool:
         """
-        Returns True if the table needs to be evaluated, and False otherwise.
+        Returns False, if the file path is sepcifically indicated as "Input file is optional and does not exist!", 
```

Comment:
Correct doc string, please. Now it doesn't rely on "Input file is optional and does not exist!" value.

Reply:
@User1000 Done!